python/pyspark/ml/feature.py

```diff
@@ -120,6 +122,200 @@ def getThreshold(self):
         return self.getOrDefault(self.threshold)
 
 
+class LSHParams(Params):
```

Comment:
It's not alphabetized here because the declaration order matters for PySpark shell.